### PR TITLE
tests, silence operations

### PR DESCRIPTION
**Pull request by RamLavi**
when running the test script, some commands give out
unnecessary log output that clatter the log
adding option is to mute these outputs

### CODE REVIEW

File: tests/test-coreos.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ fetch_latest_rhcos_image() {
   if [[ $(echo "${shasum} ${image_path}/${latest_image_gz}" | sha256sum --check) != "${image_path}/${latest_image_gz}: OK" ]]; then
     mkdir -p ${image_path}
     rm -rf ${image_path}/*
-    wget -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
+    wget -q -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
```
Comment (phoracek) [inline tests/test-coreos.sh:54]: This turns off the output completely, but I suppose it would be still interesting to see some of it. What about `-nv`?

Comment (phoracek) [reply tests/test-coreos.sh:54]:
```suggestion
    wget -nv -O ${image_path}/${latest_image_gz} ${image_url}/${latest_image_gz}
```

Comment (RamLavi) [reply tests/test-coreos.sh:54]: totally, Done

**Review by phoracek — APPROVED** [review]
Looks good! Thanks

**Pull request merged**